tests/wpunit/functions/template-tags/organizerTest.php

```diff
@@ -61,26 +61,4 @@ public function should_return_0_when_no_posts_are_found_and_found_posts_is_set()
 		$this->assertEquals( 0, $found_posts );
 	}
 
-	/**
-	 * Test multiple combinations possibles for the sensitization of organizer lists when an ordered list is presented.
-	 *
-	 * @since 4.6.15
-	 */
-	public function test_tribe_sanitize_organizers() {
```

Comment:
Why are we removing this? Instead of adjusting it to work with the new method?

Reply:
@bordoni because `tribe_sanitize_organizers` got deprecated. so is removing this test okay?